adam/curriculum_to_html.py

```diff
@@ -128,48 +128,33 @@ def _dump_instance_group(
             html_out.write(f"<head>\n\t<style>{CSS}\n\t</style>\n</head>")
             html_out.write(f"\n<body>\n\t<h1>{title} - {instance_group.name()}</h1>")
             html_out.write("\n<h2>How to read</h2>")
+            html_out.write(f"\n<p>Our curiculum is designed in two levels: The objective, here - {instance_group.name()}")
```

Comment:
What the interpolation of `instance_group.name()` here?

Reply:
@rgabbard -- copied from above to explain what the objective is (it could be replaced with "e.g. learning objects, learning roll", but it seemed more clear to me to state what the current objective is